src/MAKE/Makefile.mpi

```diff
@@ -7,12 +7,12 @@ SHELL = /bin/sh
 # specify flags and libraries needed for your compiler
 
 CC =		mpicxx
-CCFLAGS =	-g -O3 -std=c++11
+CCFLAGS =	-g -O3 # -std=c++17
```

Comment:
Why is `-std=c++17` commented out, unlike the previous?

Reply:
Because we do not yet *enforce* the C++17 requirement. As we determined from the survey of current Linux distributions and their default compilers, many (if not most) of them will already default to C++17 and they can compile LAMMPS without any problems. People with a non-C++17 compiler would get an error from the compiler if the flag was uncommented. But they would also get an error from the check in lmptype.h which has the instruction to add `-DLAMMPS_CXX11` to avoid this error.

Of course the comment will be removed *after* the next stable release when we actually will require C++17

Reply:
Got it, thanks for the explanation